server/fleet/software_installer.go

```diff
@@ -580,6 +581,11 @@ func (spec SoftwarePackageSpec) ResolveSoftwarePackagePaths(baseDir string) Soft
 	return spec
 }
 
+func (spec SoftwarePackageSpec) IncludesFieldsDisallowedInPackageFile() bool {
+	return len(spec.LabelsExcludeAny) > 0 || len(spec.LabelsIncludeAny) > 0 || len(spec.Categories) > 0 ||
+		spec.SelfService || spec.InstallDuringSetup.Valid
+}
```

Comment:
Just so I understand we aren't allowing any of these at a package level? Or we aren't allowing them at a package level if and only if they are specified at a team level?